### PR TITLE
Feature to filter submission by date

### PR DESCRIPTION
**Pull request by isht3**
`evalai challenge CHALLENGE phase PHASE submissions --start-date MM/DD/YY --end-date MM/DD/YY`

### CODE REVIEW

File: evalai/challenges.py [hunk 1]
```diff
@@ -128,14 +129,18 @@ def phase(ctx, phase):
 
 @phase.command()
 @click.pass_obj
-def submissions(ctx):
+@click.option('--start', '-s', type=Date(format='%m/%d/%y'),
```
Comment (RishabhJain2018) [inline evalai/challenges.py]: Please change it to `--start-date`

File: evalai/challenges.py [hunk 2]
```diff
@@ -128,14 +129,18 @@ def phase(ctx, phase):
...
+              help="Start date for submissions in `mm/dd/yyyy` format.")
+@click.option('--end', '-e', type=Date(format='%m/%d/%y'),
```
Comment (RishabhJain2018) [inline evalai/challenges.py]: Please change it to `--end-date`

Comment (RishabhJain2018) [inline evalai/challenges.py]: Please change it to `--end-date`

File: evalai/challenges.py [hunk 3]
```diff
@@ -128,14 +129,18 @@ def phase(ctx, phase):
...
+              help="End date for submissions in `mm/dd/yyyy` format.")
+def submissions(ctx, start, end):
```
Comment (RishabhJain2018) [inline evalai/challenges.py]: `start_date`, `end_date`

**Comment by isht3** [comment]
@RishabhJain2018 Done!

File: evalai/utils/submissions.py [hunk 4]
```diff
@@ -60,18 +63,29 @@ def pretty_print_my_submissions_data(submissions):
...
+        if (date >= start_date and date <= end_date):
+            # Check for empty method name
+            date = convert_UTC_date_to_local(submission['submitted_at'])
+            method_name = submission["method_name"] if submission["method_name"] else "None"
+            values = list(map(lambda item: submission[item], attributes))
+            values.append(date)
+            values.append(method_name)
+            table.append_row(values)
+    if len(table) == 0:
+        echo(style("\nSorry, no submissions made during this time period.\n", bold=True))
```
Comment (RishabhJain2018) [inline evalai/utils/submissions.py]: `Sorry, no submissions were made during this time period.`

Comment (isht3) [reply evalai/utils/submissions.py]: @RishabhJain2018 Updated.

**Pull request merged**